CONTRIBUTING.adoc

```diff
@@ -31,6 +29,19 @@ Testing is done differently on the desktop client and the web client.
 * The extension logs -- Go to the Command Palette and select the option 'Developer: Open Extension Logs Folder'.
 * The web view developer tools -- Go to the Command Palette and select the option 'Developer: Open Webview Developer Tools'.
 
+To run the tests suite, select 'Run Extension Tests' in the 'Run' view then, click on the green play button or run the 'Start Debbuging' command.
+
+You can also run the tests suite from a terminal using:
+
+ $ npm t
+ 
+[NOTE]
+====
+In that case, you must close your Visual Studio code instance otherwise you will get an error:
+
+ Running extension tests from the command line is currently only supported if no other instance of Code is running.
```

Comment:
```suggestion
Running extension tests from the command line is currently only supported if no other instance of Code is running.
```

Remove additional space.

Reply:
I've added a leading space to create a literal block because that's the error message you get. Maybe it's unclear? Feel free to revise my sentence 😉

Reply:
No I am just silly!